tpot/base.py

```diff
@@ -355,23 +358,36 @@ def _read_config_file(self, config_path):
                 file_string = config_file.read()
                 exec(file_string, custom_config.__dict__)
 
-            return custom_config.tpot_config
+            return custom_config
         except FileNotFoundError as e:
             raise FileNotFoundError(
                 'Could not open specified TPOT operator config file: '
                 '{}'.format(e.filename)
             )
-        except AttributeError:
-            raise AttributeError(
-                'The supplied TPOT operator config file does not contain '
-                'a dictionary named "tpot_config".'
-            )
         except Exception as e:
             raise type(e)(
                 'An error occured while attempting to read the specified '
                 'custom TPOT operator configuration file.'
             )
 
+    def _setup_pop(self, seeds, config_path):
+        """If the seeds are specified, use them as the starting population."""
+        try:
+            config = self._read_config_file(config_path)
+
+            if hasattr(config, 'seeds'):
+                seeds = config.seeds
+        except Exception as e:
+            # Config isn't a file
+            return
+
+        seed_individuals = [creator.Individual.from_string(x, self._pset) for x in seeds]
+        self._pop = []
+
+        # Add the same set of seeds to the population until we have population_size seeds
```

Comment:
With a little discussions with @rhiever earlier today, we think the `seeds` (maybe need better name for this parameter) should only specify a small part of initial population instead of initial population that is full of duplicated pipelines in the `seeds` (If `seeds` is a small list, then the pipeline diversity of GP will be limited in the beginning).  Other pipelines in initial population rather than pipelines in `seeds` should be randomly generated.